modules/dacIdSystem.md

```diff
@@ -0,0 +1,25 @@
+## DAC User ID Submodule
+
+## Building Prebid with DAC ID Support
+
+First, make sure to add the DAC ID submodule to your Prebid.js package with:
```

Comment:
This doc is fine, but it may be valuable to explain any other dependencies to use your module. Based on the code, it seems like separate code needs to be in place for this module to pick up your ID. So you may want to talk about that requirement and how to get in touch with you if the publisher would like to work with you.

Not a requirement, just a suggestion without knowing your business model or anything.

Reply:
Thank you for comment.
I'll add document.